src/components/navigation-header/index.tsx

```diff
@@ -104,7 +108,14 @@ const NavigationHeader = () => {
 		<header className={s.root}>
 			<LeftSideHeaderContent />
 			<div className={s.rightSide}>
-				<GiveFeedbackButton />
+				{GLOBAL_SEARCH_ENABLED ? (
+					<CommandBarActivator
+						leadingIcon={<IconSearch16 />}
+						visualLabel="Search"
+					/>
+				) : (
+					<GiveFeedbackButton />
```

Comment:
🗒️ The feedback mechanism will be moved to the footer of the command bar, so that's why `GiveFeedbackButton` is now being conditionally rendered.